tests/openshift/templates/bddstack.bc.yaml

```diff
@@ -3,123 +3,169 @@
-              INSTALL_PKGS="rh-python37 rh-python37-python-devel rh-python37-python-setuptools rh-python37-python-pip \
-              nss_wrapper httpd24 httpd24-httpd-devel httpd24-mod_ssl httpd24-mod_auth_kerb httpd24-mod_ldap httpd24-mod_session \
-              atlas-devel gcc-gfortran libffi-devel libtool-ltdl rh-nodejs8 rh-nodejs8-nodejs-nodemon unzip \
-              java-11-openjdk java-11-openjdk-devel redhat-lsb libXScrnSaver gdk-pixbuf2 xorg-x11-server-Xvfb wget" && \
-              ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon && \
-              yum remove -y $UNINSTALL_PKGS &&\
-              yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
-              rpm -V $INSTALL_PKGS && \
-              yum clean all -y && \
-              # Install Gradle
-              mkdir /opt/gradle && \
-              wget https://services.gradle.org/distributions/gradle-4.2.1-bin.zip -P /opt/gradle && \
-              unzip -d /opt/gradle gradle-4.2.1-bin.zip && \
-              rm /opt/gradle gradle-4.2.1-bin.zip && \
-              # Get Google Chrome latest version
-              wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm && \
-              yum -y localinstall google-chrome-stable_current_x86_64.rpm && \
-              rm google-chrome-stable_current_x86_64.rpm && \
-              if [ ! -d /home/jenkins/.pki ] ; then mkdir /home/jenkins/.pki; fi && \
-              chmod 777 /home/jenkins/.pki
-
-          # In order to drop the root user, we have to make some directories world
-          # writable as OpenShift default security model is to run the container under
-          # random UID.
-          RUN mkdir -p /opt/app-root  && \
-              chown -R 1001:0 /opt/app-root && \
-              chmod -R og+rwx /opt/app-root && \
-              # Obtain the installed Chrome version so that BDDStack can load the correct driver by querying the chromeversion and chromedriver files
-              google-chrome --version | grep -o -m 1 -E "[0-9]{2,3}[^ ^a-z^A-Z^0-9^\.]*" | sed -n 1p >  /opt/app-root/chromeversion && \
-              wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE_`cat /opt/app-root/chromeversion` -O  /opt/app-root/chromedriver
-
-          USER 1001
-
-        git:
-          ref: Testing-and-Scanning-Swiss-Army-Knife-Slave
-          uri: "https://github.com/rstens/openshift-tools"
-        type: Git
-      strategy:
-        dockerStrategy:
-          env:
-            - name: OPENSHIFT_JENKINS_JVM_ARCH
-              value: x86_64
-          from:
-            kind: ImageStreamTag
-            name: jenkins-slave-base-rhel7:latest
-        type: Docker
-      triggers:
-        - type: ConfigChange
-        - imageChange: {}
-          type: ImageChange
+spec:
+  failedBuildsHistoryLimit: 5
+  nodeSelector: null
+  output:
+    to:
+      kind: ImageStreamTag
+      name: 'bddstack:latest'
+  postCommit: {}
+  resources: {}
```

Comment:
If you don't want this to take 1+ hour to build, you need to put in explicit resource requests and limits. I'd recommend the following (with a temporal cutoff):
```yaml
      completionDeadlineSeconds: 600
      resources:
        requests:
          cpu: 2000m
          memory: 1Gi
        limits:
          cpu: 4000m
          memory: 4Gi
```

Reply:
6 minutes build now. So not an issue anymore.